doc/src/cylc-user-guide/job-host.tex

```diff
@@ -1,88 +1,356 @@
-\section{Remote Job Host Interaction}
+\section{Remote Job Management}
 
-\lstset{language=jobhosts}
+Managing tasks in a workflow requires more than just job execution: cylc
+performs additional actions with \lstinline=rsync= for file transfer, and
+direct execution of \lstinline=cylc= sub-commands over non-interactive
+SSH.\footnote{Cylc used to run bare shell expressions over SSH, which required
+a bash shell and made whitelisting difficult.} This section shows exactly what
+cylc does on and to remote job hosts.
 
-This appendix shows transcripts of all \lstinline=ssh=, \lstinline=scp=, and
-\lstinline=rsync= invocations from a Cylc suite called \lstinline=test= started by
-\lstinline=rose suite-run= on \lstinline=vagrant@localhost=. The suite
-submits a single task called \lstinline=foo= to \lstinline=hobo@otherhost=.
-The operator polls then kills the task, then shuts the suite down.
+Security-conscious sites may want to restrict access to job hosts with SSH
```

Comment:
Can we tone this down slightly? It is not that other sites are not security conscious. It is that sites may have different security audit requirements.